proto/redpanda/api/dataplane/v1alpha1/kafka_connect.proto

```diff
@@ -60,36 +55,42 @@ message ConnectCluster {
 
 message ConnectorStatus {
   message Connector {
+    // State of the connector instance.
     string state = 1;
     string worker_id = 2;
     string trace = 3;
   }
+  // Name of connector.
   string name = 1;
   Connector connector = 2;
+  // Status of connector tasks. For more information, see the [https://docs.redpanda.com/current/deploy/deployment-option/cloud/managed-connectors/monitor-connectors/#connector-tasks](Monitor Connectors) documentation.
   repeated TaskStatus tasks = 3;
   string type = 4;
-  // holistic_state of all the tasks within the connector this is our internal
-  // holistic state concept
+  // State of the connector including all the tasks within the connector.
   ConnectorHolisticState holistic_state = 5;
-  // Errors is list of parsed connectors' and tasks' errors
+  // List of parsed connectors' and tasks' errors.
   repeated ConnectorError errors = 6;
 }
 
 message TaskStatus {
+  // ID of connector task.
   int32 id = 1;
+  // State of connector task.
   string state = 2;
   string worker_id = 3;
   string trace = 4;
 }
 
 message TaskInfo {
+  // Name of connector.
   string connector = 1;
   int32 task = 2;
```

Comment:
Could we add a description for this field?

Reply:
```suggestion
  // The connector task ID.
  int32 task = 2;
```